app/actions/index.js

```diff
@@ -125,7 +168,7 @@ export const clearRemoteSurveys = () => (dispatch, getState) =>
     type: types.CLEAR_REMOTE_SURVEYS
   });
 
-export const fetchRemoteSurvey = (id, url) => (dispatch, getState) => {
+export const fetchRemoteSurvey = (id, url, target) => (dispatch, getState) => {
```

Comment:
Survey defs are probably not specific to certain bounding areas--they should apply to any data loaded onto the device (so: AOIs different from survey definitions).